src/USER-SCAFACOS/fix_scafacos.cpp

```diff
@@ -439,15 +439,15 @@ bool FixScafacos::check_result(FCSResult result, int comm_rank)
 {
   if (result) 
   {
-    printf("ScaFaCoS Error: Caught error on task %d.\n", comm_rank);
+    fprintf(stdout,"ScaFaCoS Error: Caught error on task %d.\n", comm_rank);
```

Comment:
Ok I think the easiest way is to remove the fprintf statement and just pass the internal error message to the error -> one() call. As it is possible that a subset of processes (>1) may encounter an error, is error -> one() still the correct call?

Reply:
>As it is possible that a subset of processes (>1) may encounter an error, is error -> one() still the correct call?

Yes this is fine. You can only use `error->all` if you can guarantee that all processes will reach it, otherwise it will deadlock.